<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Abhishek" ignored="1" name="Personal Loan" tests="5" failures="1" timestamp="2021-05-15T18:41:28 IST" time="0.152" errors="0">
  <testcase name="Demo" time="0.013" classname="test.day1"/>
  <testcase name="ploan" time="0.009" classname="test.day2"/>
  <testcase name="LoginAPIHomeLoan" time="0.006" classname="test.day4"/>
  <testcase name="MobileloginHomeLoan" time="0.009" classname="test.day4">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day4.MobileloginHomeLoan(day4.java:21)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- MobileloginHomeLoan -->
  <testcase name="webLoginHomeLoan" time="0.004" classname="test.day4"/>
  <testcase name="SecondTest" time="0.0" classname="test.day1">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal Loan -->
